### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by AntonellaMorittu**
#### Description
Changes proposed in this pull request:
* Added general information and description
* Added  technologies  and tests used
* Added authors and contributors
#### What I have learned working on this feature: 
How to write an application README from scratch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,142 @@
...
+
+## Contributors
+
+Our restless coaches supported us in this project:
+
+* Thomas Ochman https://github.com/tochman
+* Raoul Diffouo https://github.com/diraulo
+* Faraz Naeem https://github.com/faraznaeem
+* Magnus Þór Ársælsson https://github.com/magnus-thor
+* Olver Ochman ...
```
Comment (isabellelii) [inline README.md]: Change Olver to Oliver

Comment (kmthorsnes) [inline README.md]: Add github for him too ?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,142 @@
...
+
+Welcome to Pap News - _Spread the knowledge!_
+
+## Description
+Week 8 Project of Craft Academy cohort January 2018.
+
+PapsNews is a newsagency Web Application used to share your local and international news directly on your device screen.  
+
+Once you land on the Homepage (https://....herokuapp.com/), you need to Signup in order to have access to the web contents
+(click on sign up in the navbar), you will automatically have 7 days free trial, at the end of which you can become a subscriber for a minimum period of 12 months.
```
Comment (kmthorsnes) [inline README.md]: Are we sure about this trial functionality?

Comment (NellaMori) [reply README.md]: We agreed this morning, remember? Anyway, it's a work in progress document, it will be updated everyday.

File: README.md (hunk 1 above)
Comment (NellaMori) [reply README.md]: I removed the incomplete info, it will be added tomorrow.

File: README.md (hunk 2 above)
Comment (kmthorsnes) [reply README.md]: OK, I must have misunderstood something, No probs.

**Comment by diraulo** [comment]
@AntonellaMorittu As I was saying in Scrum earlier, the reason your PR is failing right now is because there's no `.travis.yml` file on this branch. So pulling code from `develop` onto this should get this to pas again. 

When things aren't passing with CI, click the `details` link to check the logs on Travis or whatever CI tool you're using to get an understanding of what went wrong... and ultimately getting a solution for it 🙂

**Review by AKidd95 — APPROVED** [review]
Looks good so far

**Comment by NellaMori** [comment]
@diraulo yes I know, I was just triple checking you would have been okay with it ..you guys are unpredictable :)

**Review by isabellelii — APPROVED** [review]

**Review by kmthorsnes — APPROVED** [review]

**Pull request merged**